packages/core/src/hooks/data/useCreateMany.spec.tsx

```diff
@@ -346,4 +346,48 @@ describe("useCreateMany Hook", () => {
             });
         });
     });
+
+    it("correctly with `interval` and `onInterval` params", async () => {
+        const onInterval = jest.fn();
+        const { result } = renderHook(
+            () =>
+                useCreateMany({
+                    overtimeOptions: {
+                        interval: 100,
+                        onInterval,
+                    },
+                }),
+            {
+                wrapper: TestWrapper({
+                    dataProvider: {
+                        default: {
+                            ...MockJSONServer.default,
+                            createMany: () => {
+                                return new Promise((res) => {
+                                    setTimeout(() => res({} as any), 1000);
+                                });
+                            },
+                        },
+                    },
+                    resources: [{ name: "posts" }],
+                }),
+            },
+        );
+
+        result.current.mutate({
+            resource: "posts",
+            values: [{ title: "foo" }, { title: "bar" }],
+        });
+
+        await waitFor(() => {
+            expect(result.current.isLoading).toBeTruthy();
+            expect(result.current.overtime.elapsedTime).toBe(900);
+            expect(onInterval).toBeCalled();
+        });
+
+        await waitFor(() => {
+            expect(!result.current.isLoading).toBeTruthy();
```

Comment:
I think it's hard to read.

```suggestion
            expect(result.current.isLoading).toBeFalsy();
```